polish docs (#23)
apache/skywalking-satellite: docs/en/guides/compile/compile.md

> # Compile

Reviewer: Compile -> Compiling

Don't use a verb in the document.

## Platform

> Linux and MacOs is supported in SkyWalking Satellite. Windows is not good supported beacuse some features is not adaptive on the Windows, such as the mmap feature. If you want to run it on the windows platform, please read [the doc](../../FAQ/running_on_windows.md).

Reviewer: ```suggestion
Linux and MacOS are supported in SkyWalking Satellite. Windows is not due to following components please read [the doc](../../FAQ/running_on_windows.md).
```

> The Windows platform does not support plugins list:

Reviewer: ```suggestion
```

# Compiling

## Platform

> Linux and MacOS are supported in SkyWalking Satellite. Windows is not due to following components. 

Reviewer 2: Is Windows unsupported anyway, or do we have a fallback queue implementation on Windows? For example, users can try it on Windows with `memory-queue`?

Author: already push a memory queue

Reviewer 2: > already push a memory queue

I know that, I mean we cannot simply say "Windows is not due to following components.", `mmap-queue` is only a component in Satellite, users can still try / use Satellite if they configure another component (like `memory-queue`), the unavailability is component level and should be documented in that specific component, i.e. document "This component is not available on Windows" in [the `mmap-queue` plugin doc](https://github.com/apache/skywalking-satellite/blob/main/docs/plugins/queue/mmap/README.md), instead of here, "compile" doc.

Reviewer: This is the question in my mind, if memory queue is available, then satellite should be available for WIndows platform. We should be clear about that. Could you deliver the window version binary? The current idea is saying code change, which is not a good one.

Author: fixed

Author: zhenxu help me to unify the compile at all platforms. Thanks a lot.
